### PR TITLE
Shop attributes added

### PR DESCRIPTION
**Pull request by larsbo**
### 1. Why is this change necessary?
Because everything is better with ~~~bacon~~~ attributes!

### 2. What does this change do, exactly?
It adds the possibility to add attributes (`Freitextfelder`) to the shop entity.

### 3. Describe each step to reproduce the issue or behaviour.

After merging this PR you can create shop attributes in the backend attribute module or inside plugins:
attribute config

Attributes can be set in the shop configuration:
shop attributes

and used in the frontend:
attributes frontend

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: _sql/migrations/1441-add-shop-attributes-table.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public function up($modus)
+    {
+        $sql = <<<'SQL'
+CREATE TABLE IF NOT EXISTS `s_core_shops_attributes` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `shopID` int(11) UNSIGNED NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE INDEX `shopID` (`shopID`),
+  CONSTRAINT `FK__s_core_shops` FOREIGN KEY (`shopID`) REFERENCES `s_core_shops` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE
+) ENGINE=InnoDB;
```
Comment (shyim) [inline _sql/migrations/1441-add-shop-attributes-table.php]: Please add DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci

**Comment by shyim** [comment]
The Shopware\Bundle\StoreFrontBundle\Gateway\DBAL\ShopGateway does not consider the new attributes.

**Comment by larsbo** [comment]
@shyim I hope I didn't miss anything 😊

File: UPGRADE-5.5.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ This changelog references changes done in Shopware 5.5 patch versions.
 
 ### Changes
 
+* Added attributes to Shop configuration
```
Comment (mitelg) [inline UPGRADE-5.5.md:19]: if we add something new, this would go under another headline, have a look at the other upgrade files :wink:

Comment (larsbo) [reply UPGRADE-5.5.md:19]: damn, you're right! 🙆‍♂️

Comment (larsbo) [reply UPGRADE-5.5.md:19]: I switched the order of the headlines to match the schema of the other versions.

**Pull request closed without merging**